Dockerfile

```diff
@@ -4,18 +4,22 @@ FROM node:alpine
 # Setting the working directory for the application
 WORKDIR /app
 
+# Install serve package
+RUN npm install -g serve
+
```

Comment:
By installing this beforehand, we reduce the developer's time when we rebuild this container

Reply:
Confirmed!